entproto/cmd/protoc-gen-entgrpc/converter.go

```diff
@@ -90,29 +92,67 @@ func (g *serviceGenerator) newConverter(fld *entproto.FieldMappingDescriptor) (*
 	}
 	return out, nil
 }
+func convertPbMessageType(fqn, entFieldType string, conv *converter) error {
+	switch {
+	case fqn == "google.protobuf.Timestamp":
+		conv.toProtoConstructor = protogen.GoImportPath("google.golang.org/protobuf/types/known/timestamppb").Ident("New")
+	case isWrapperType(fqn):
```

Comment:
```suggestion
func convertPbMessageType(fd *desc.FieldDescriptor, ft *field.TypeInfo, conv *converter) error {
	switch m := GetMessageType().GetFullyQualifiedName(); {
```

Why passing strings and not the concrete types? You lose the types when you pass strings.

Reply:
good point